sample/src/extension.ts

```diff
@@ -264,6 +264,42 @@ ${stderr}`);
 ${JSON.stringify(result) ?? 'undefined'}`);
     });
 
+    const sampleAvailableInstallsRegistration = vscode.commands.registerCommand('sample.dotnet.availableInstalls', async (version) =>
+    {
+        let dotnetPath = await vscode.window.showInputBox({
+            placeHolder: 'undefined',
+            value: 'undefined',
+            prompt: 'The .NET Host Path to Scan.',
+        });
+
+        dotnetPath = dotnetPath === 'undefined' ? undefined : dotnetPath;
+
+        let arch = await vscode.window.showInputBox({
+            placeHolder: 'x64',
+            value: 'x64',
+            prompt: 'The .NET runtime architecture.',
+        });
+
+        arch = arch?.toLowerCase();
+
+        let searchMode = await vscode.window.showInputBox({
+            placeHolder: 'runtime',
+            value: 'runtime',
+            prompt: 'look for an sdk, runtime, aspnetcore runtime, etc',
+        });
+
+        try
+        {
+            const result = await vscode.commands.executeCommand<IDotnetAcquireResult>('dotnet.availableInstalls', { architecture: arch, requestingExtensionId: requestingExtensionId, mode: searchMode, dotnetExecutablePath: dotnetPath });
```

Comment:
If `dotnetPath` is undefined, what happens here?

Reply:
'dotnet' is used instead by the API; good question. We need to support that based on the IDotnetSearchContext specification.